docs/content/docs/_index.md

```diff
@@ -304,11 +304,11 @@ The priority of operations is the following, from lowest to highest:
 
 - `and`: true if the left and right operands are true
 - `or`: true if the left or right operands are true
-- `not`: negate a statement
+- `not`: negate an expression
 
-#### String concatenation
+#### Concatenation
 
-You can concatenate several strings/idents using the `~` operator.
+You can concatenate several strings/numbers/idents using the `~` operator.
```

Comment:
function calls as well, provided they return a string or a number

Reply:
I was thinking about this. Shouldn't function calls be allowed to replace idents if function returns the result of type same as that of ident?

I mean, function calls should be able to replace variables/idents.

e.g.
````
{{ 5 in [0, 1, 2, 3, 4, 5, 6] }}
````
This should be equivalent to:
```
{{ 5 in range(end=7) }}
```
Does parser v2 handles this? Currently it's a parser error.

If a variable is allowed then function returning that type of result is too. So there will be no need to mention 'function calls too' alongside `idents` in docs.

Reply:
> Does parser v2 handles this?

Yep

Reply:
Cool! Then we don't need to mention "function calls as well" since one can always replace idents with function calls so only mentioning "idents" is enough.

Otherwise we'll have to accompany "idents" with "function calls as well" all over the doc.